components/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt/src/test/java/org/wso2/carbon/identity/api/resource/mgt/dao/CacheBackedAPIResourceManagementDAOTest.java

```diff
@@ -529,6 +576,7 @@ private APIResource addAPIResourceToDB(String namePostFix, Connection connection
                     connection.commit();
                     return null;
                 });
+        organizationManagementUtil.when(() -> OrganizationManagementUtil.isOrganization(anyInt())).thenReturn(false);
```

Comment:
Why this mocking is required? can't find a usage of this in the DAO impl of addAPIResource method

Reply:
When adding the API resource, inside the DAO method, it will extract the added API resource from the id[1]. Inside that method we added the organization checkl and this mocking is needed for that.

<img width="1013" alt="image" src="https://github.com/user-attachments/assets/b8e4560d-2e52-4887-8d40-1e9467f3a1ff">


[1] https://github.com/wso2/carbon-identity-framework/blob/0033577bc92ba41864b8cf0f978073c2900f2fe2/components/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt/src/main/java/org/wso2/carbon/identity/api/resource/mgt/dao/impl/APIResourceManagementDAOImpl.java#L168